README.md

```diff
@@ -0,0 +1,102 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To do List] <a name="about-project"></a>
+
+**A simple To do List**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="">linters</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+    <li><a href="https://github.com">GIT/Github</a></li>
+    <li><a href="https://nodejs.org/en/">NodeJS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **[Dynamic To Do List]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](comming soon)
+
+## Loom Video <a name="loom"></a>
+- [Video Link](comming soon)
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- VSCode or any other IDE.
+- node.js(preferable).
+- browser to deploy.
+- GitHub/Git.
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+git clone https://github.com/Rudy-Menekam/to-do-list.git
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+- GitHub: [@githubhandle](https://github.com/Rudy-Menekam)
+- Twitter: [@twitterhandle](https://twitter.com/MenekamR)
+- LinkedIn: [linkedIn](https://www.linkedin.com/in/menekam-rudy/) 
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Refactor code to add and remove to/from list**
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Rudy-Menekam/to-do-list/issues).
+
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you are welcomed to star and fork it. You can also share your ideas for future features. 
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the todo list project.
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- Hey, great job adding the readme file, however, it is not following the [template](https://github.com/microverseinc/readme-template) as supposed to. please, kindly update the readme to follow the structure as seen in the template. Also, kindly remove all the placeholder text as these texts are there to guide on how to add the correct content for the appropriate section to make the readme more professional.